.jenkins/continuous.groovy

```diff
@@ -447,14 +447,14 @@ pipeline {
                         }
                     }
                 }
-                stage('CUDA-11.8-Clang-15') {
+                stage('CUDA-11.8-Clang-16') {
                     agent {
                         dockerfile {
                             filename 'Dockerfile.nvcc'
                             dir 'scripts/docker'
                             label 'nvidia-docker && volta'
                             args '-v /tmp/ccache.kokkos:/tmp/ccache --env NVIDIA_VISIBLE_DEVICES=$NVIDIA_VISIBLE_DEVICES'
-                            additionalBuildArgs '--build-arg BASE=nvcr.io/nvidia/cuda:11.8.0-devel-ubuntu22.04 --build-arg ADDITIONAL_PACKAGES="clang-15 clang-tidy-15"'
+                            additionalBuildArgs '--build-arg BASE=nvcr.io/nvidia/cuda:11.8.0-devel-ubuntu22.04 --build-arg ADDITIONAL_PACKAGES="clang-16 clang-tidy-16"'
```

Comment:
AFAIK there is no `clang[-tidy]-16` package.
I'd recommend keeping that build as is for now